lib/clients/incoming-webhook/client.js

```diff
@@ -14,6 +14,7 @@ var noop = require('lodash').noop;
  * @param {string} defaults.channel The default channel to use when sending a webhook.
  *      If no channel is specified, the one chosen when creating the webhook will be used.
  * @param {string} defaults.text The default text to use when sending a webhook.
+ * @param {string} defaults.link_names Do not link names and mention as default.
```

Comment:
i think this line is wrong. the parameter name should be `defaults.linkNames` (notice the camel case). and the description isn't clear. I suggest: "The default setting for the link_names format option to use when sending a webhook. If no value is specified, the one chosen when creating the webhook will be used."